pkg/sql/crdb_internal.go

```diff
@@ -1131,6 +1134,40 @@ CREATE TABLE crdb_internal.session_trace (
 	},
 }
 
+// crdbInternalInflightTracingTable exposes the node-local registry of in-flight spans.
+var crdbInternalInflightTracingTable = virtualSchemaTable{
+	comment: `in-flight spans (RAM; local node only)`,
+	schema: `
+CREATE TABLE crdb_internal.node_inflight_tracing (
+  span_id    INT NOT NULL,        -- The span's ID.
+  trace_id   INT NOT NULL,        -- The trace's ID.
+  duration   INTERVAL             -- The span's duration, measured by time of 
+                                  -- collection - start time for all in-flight spans.
+)`,
+	populate: func(ctx context.Context, p *planner, _ *dbdesc.Immutable, addRow func(...tree.Datum) error) error {
+		p.ExecCfg().Settings.Tracer.VisitSpans(func(span *tracing.Span) error {
```

Comment:
CI is failing on this line with the error

_[Test archive] pkg/sql/crdb_internal.go:1148:42: cannot use func literal (type func(*tracing.Span) error) as type func(*tracing.Span) in argument to p.ExecCfg().Settings.Tracer.VisitSpans_

It seems to compile fine locally and in fact complains if I remove the `error`. Per discussion with @knz I added `error` to the `VisitSpans` API and have added it as an argument where `VisitSpans` is used (including here).